source/documentation/joiners_leavers.md

```diff
@@ -0,0 +1,73 @@
+# Joiners & Leavers
+
+This should be a complete list of tools/services that new technology team
+joiners should be given access to, and leavers be removed from.
+
+
+## Joiners checklist
+
+### Try it out!
+
+1. [Sign up to GovWifi!](https://www.wifi.service.gov.uk/about-govwifi/connect-to-govwifi/)
+2. Ask your tech lead to get added to the [Admin Portal](https://admin.wifi.service.gov.uk/) and [Admin Portal - Staging](https://admin.staging.wifi.service.gov.uk/)
+
+### Infrastructure
+
+1. [Request an AWS account](https://gds-request-an-aws-account.cloudapps.digital/).
+2. Ask RE:D to add you to GovWifi account(s).
```

Comment:
is it common knowledge how to get to these people?

If not, maybe reference `#re-d@slack`